fomantic/Fomantic-UI
# fix(dropdown): observe select attribute changes

## Description
The select observer in dropdown did not detect attribute changes  of `option` or `optgroup` changes if the dropdown was build out of a `select` tag

## Testcase
- Open the dropdown and see an option group "Swedish Cars" and the "Rabbit" entry being available
- Close the dropdown
- Click on the "Change select tag" button
- Open the Dropdown again
 
## Broken
- Nothing has changed
https://jsfiddle.net/lubber/hjdv6bnL/8/

## Fixed
- "Swedish Cars" changed into "I was changed"
- "Rabbit" is disabled
https://jsfiddle.net/lubber/hjdv6bnL/9/


## Screenshot
|Broken|Fixed|
|-|-|
|![selectobserverbroken](https://user-images.githubusercontent.com/18379884/127381609-3cf3a29a-2d9d-43dc-8802-7fd1d50cd6b8.gif)|![selectobserverfixed](https://user-images.githubusercontent.com/18379884/127381621-2cef07b0-fcc3-4ae3-a4b7-4205087fd8da.gif)|

## Closes
#2033
